rosidl_generator_c/resource/msg__functions.c.em

```diff
@@ -20,6 +20,7 @@ from rosidl_generator_c import value_to_c
 
 msg_typename = '%s__%s__%s' % (spec.base_type.pkg_name, subfolder, spec.base_type.type)
 array_typename = '%s__Array' % msg_typename
+cleanup_lines = []
```

Comment:
Please move this closer to where it is actually being used. Likely under the definition of `abort_lines`.